packages/models/src/store/store.ts

```diff
@@ -86,6 +88,24 @@ export class Store<
     this.#type = Reflect.getMetadata(ProviderKey.TypePattern, this.constructor)
   }
 
+  public load(path?: string) {
+    let local = {}
+    const customizer = (objValue: StructSchema, srcValue: StructSchema) => {
+      if (Array.isArray(objValue)) {
```

Comment:
Shall we sum values if their type is BigNumber or BI or number? BI or number is a type of molecule

Reply:
It's feasible, but the rule should be considered and documented explicitly, it can be a feature in the future.

Now, the duplicated field will be overridden by the latter one.